multiversx_sdk/core/transaction_computer.py

```diff
@@ -34,10 +34,10 @@ def compute_transaction_fee(self, transaction: ITransaction, network_config: INe
 
         return int(fee_for_move + processing_fee)
 
-    def compute_bytes_for_signing(self, transaction: ITransaction) -> bytes:
+    def compute_bytes_for_signing(self, transaction: ITransaction, with_signature: bool = False) -> bytes:
```

Comment:
this method should not have the extra `with_signature` parameter. This param is needed only for the `_to_dictionary()` method.

Reply:
you are right. changed.